src/Control/Effect/Catch.hs

```diff
@@ -13,18 +13,17 @@ module Control.Effect.Catch
   ( Catch (..)
   , catch
   , catchSync
-  , runCatch
-  , CatchC (..)
   ) where
 
-import           Control.Effect.Carrier
+import           Control.Carrier
 import           Control.Effect.Reader
 import           Control.Effect.Sum
 import qualified Control.Exception as Exc
 import           Control.Exception.Safe (isSyncException)
 import           Control.Monad.IO.Class
 import           Control.Monad.IO.Unlift
 
+-- | @since 0.1.0.0
 data Catch m k
```

Comment:
I kinda feel like we should rename this to `CatchIO`. Thoughts?

Reply:
Ehh, I could go either way. Let’s think about it. I’m gonna merge this in the meantime.